### ISSUE TITLE
Adding one element to a cons

### ISSUE THREAD

**Issue by Munksgaard** [hrldcpr/elm-cons#6]
It would be nice if there was a builtin function for pushing a new element on top of the Cons:

```elm
push : a -> Cons a -> Cons a
```

If course, the functionality is easily achievable through other means, but it seems like such a base function that it's a shame it's not included by default.

**Comment by hrldcpr** [comment]
Great point! And I think `push` is a good name, given that it's essentially a stack. I'll add this soon.

```elm
push : a -> Cons a -> Cons a
push first rest = consWithMaybe first <| Just rest
```

**Comment by hrldcpr** [comment]
Released as 3.1.0 (supposedly… not sure how long it takes). Thanks again!

**Comment by Munksgaard** [comment]
Thank you for reacting so quickly, and for a nice package :+1:
